dayone.hs

```diff
@@ -21,28 +21,28 @@
 
 -- Part  1:
 -- change next line: add parentheses to make this equal "Functional"
-word1 = reverse "lanoi" ++ "tcnuF"    -- "++"concatenates concatenates strings
+word1 = reverse ("lanoi" ++ "tcnuF")    -- "++"concatenates concatenates strings
   
 
 incr n = n + 1 -- don't change this...
 
 -- Part 2:
 -- change next line to make equal 50
-fifty = incr 7 * 7
+fifty = incr (7 * 7) --just parentheses
 
 
 -- Part 3:
 -- this function adds 2 to its parameter. Change it to use `incr`
 -- twice, rather than using `+ 2`
-add2 n = n + 2
-
+add2 n = incr(incr n) --recursion but based
 
 -- Part 4:
 -- You could also write `add2` as:
 newAdd2 = incr . incr
 --
 -- Use the 'net to find out what the "." operator is called
--- and write it here: «your answer»
+-- and write it here: 
+-- The "." operator allows you to chain functions by putting the output of the fnction to the right as the input of the function to the left. Less parentheses I guess.
```

Comment:
Yup. But it's more than just saving punctuation. What you're really doing is creating a brand new function 
composed of the original two. `f(g(x))` has two functions that happen to be able to be used together. `f . g` is a new function with a specific purpose.

Underneath, the effect may be the same. But thinking of it as composition can change the way you write the code.